cachi2/core/models/input.py

```diff
@@ -73,6 +73,7 @@ class BundlerPackageInput(_PackageInputBase):
     """Accepted input for a bundler package."""
 
```

Comment:
**Commit message:**

If this patch is supposed to resolve the issue in question then it should be prefixed with one of the magic words (e.g. Resolves) to automatically close the issue on merge.